Gemfile

```diff
@@ -4,7 +4,6 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 gemspec
 
 platform :jruby do
-  gem 'activerecord-jdbcsqlite3-adapter', github: 'jruby/activerecord-jdbc-adapter', branch: 'rails-5'
```

Comment:
I think including something in `Gemfile` is still necessary so that 5.0 and 5.1 get this gem, but you probably don't need to pin it to a particular version.

Reply:
So just `gem 'activerecord-jdbcsqlite3-adapter'` should work, and then `bundle exec appraisal install`